dashboard/test/test_helper.rb

```diff
@@ -229,6 +229,13 @@ class ActionController::TestCase
     request.env['cdo.locale'] = 'en-US'
   end
 
+  # As `current_user` is not accessible from controller tests (only from within the controller),
+  # the signed in user is only accessible from the session.
+  # @returns [Integer, nil] The ID of the signed in user, nil if no user is signed in.
+  def signed_in_user_id
```

Comment:
As `current_user` is separate from the "session" user, I don't want to name this `current_user_id`. Is `signed_in_user_id` reasonable?